src/Cli/ServeCommand.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Antidot\Framework\Cli;
+
+use React\ChildProcess\Process;
+use React\EventLoop\Loop;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class ServeCommand extends Command
+{
+    const NAME = 'serve';
```

Comment:
Maybe it could define the visibility:
```suggestion
    public const NAME = 'serve';
```